src/Pages/Writing/Writing.js

```diff
@@ -0,0 +1,316 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import CategoryBox from './components/CategoryBox';
+import UploadPhoto from './components/UploadPhoto';
+import { Link } from 'react-router-dom';
+
+function Writing() {
+  const [isDroped, setIsDroped] = useState('');
+  const [isChanged, setIsChanged] = useState([]);
+  const [size, setSize] = useState('평수');
+  const [type, setType] = useState('주거형태');
+  const [style, setStyle] = useState('스타일');
+  const [roomType, setRoomType] = useState('공간');
+  const [imgFile, setImgFile] = useState('');
+  const [previewUrl, setPreviewUrl] = useState('');
+  const [textData, setTextData] = useState('');
+
+  useEffect(() => {
+    for (let i = 0; i < 4; i++) {
+      isChanged.push(0);
+    }
+  }, [isChanged]);
+
+  const handleDrop = idx => {
+    setIsDroped(idx);
+    isChanged[idx] = 1;
+    setIsChanged(isChanged);
+    console.log(isChanged, '1');
+  };
+
+  const chooseSize = e => {
+    setIsDroped(false);
+    setSize(e.target.value);
+  };
+  const chooseType = e => {
+    setIsDroped(false);
+    setType(e.target.value);
+  };
+  const chooseStyle = e => {
+    setIsDroped(false);
+    setStyle(e.target.value);
+  };
+  const chooseRoomType = e => {
+    setIsDroped(false);
+    setRoomType(e.target.value);
+  };
+
+  const showImg = e => {
+    e.preventDefault();
+    let reader = new FileReader();
+    let file = e.target.files[0];
+
+    reader.onloadend = () => {
+      setImgFile(file);
+      setPreviewUrl(reader.result);
+    };
+    reader.readAsDataURL(file);
+  };
+
+  const handleText = e => {
+    setTextData(e.target.value);
+  };
+
+  const handleSubmit = () => {
+    const sendObject = {
+      size: size,
+      type: type,
+      style: style,
+      roomType: roomType,
+      text: textData,
+    };
```

Comment:
```suggestion
    const sendObject = {
      size,
      type,
      style,
      roomType: roomType,
      text: textData,
    };
```

위와 같이 key와 value 값이 같을 때 단축할 수 있습니다!
그리고 해당 객체 안에 업로드 된 이미지 정보도 들어가야 겠네요~

Reply:
수정하였습니다!